premium/eagle-eye/crowd-eagle-eye/__init__.py

```diff
@@ -0,0 +1,2 @@
+if __name__ == '__main__':
```

Comment:
Nit: do we need this?

Reply:
Aah no - this is what pycharm does for some reason and I didn't notice it when I committed changes to eagle eye with the premium stuff.